<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabControl" >
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>Connection</string>
       </attribute>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QGroupBox" name="optionsGrouBox" >
          <property name="title" >
           <string>Options</string>
          </property>
          <layout class="QGridLayout" >
           <property name="margin" >
            <number>9</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item row="0" column="0" >
            <widget class="QLabel" name="label" >
             <property name="text" >
              <string>Port:</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="buddy" >
              <cstring>PortComboBox</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QLabel" name="label_2" >
             <property name="text" >
              <string>LogSpec:</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="buddy" >
              <cstring>LogSpecComboBox</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="1" >
            <widget class="QComboBox" name="LogSpecComboBox" />
           </item>
           <item row="0" column="1" >
            <widget class="QComboBox" name="PortComboBox" />
           </item>
           <item row="0" column="2" >
            <widget class="QLabel" name="label_3" >
             <property name="text" >
              <string>Baud Rate:</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="buddy" >
              <cstring>BaudRateBox</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="3" >
            <widget class="QSpinBox" name="BaudRateBox" >
             <property name="maximum" >
              <number>115200</number>
             </property>
             <property name="minimum" >
              <number>1200</number>
             </property>
             <property name="value" >
              <number>15625</number>
             </property>
            </widget>
           </item>
           <item row="1" column="2" >
            <widget class="QPushButton" name="specReloadButton" >
             <property name="text" >
              <string>Reload</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2" >
          <property name="title" >
           <string>Status</string>
          </property>
          <layout class="QGridLayout" >
           <property name="margin" >
            <number>9</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item row="1" column="0" colspan="2" >
            <widget class="QTextBrowser" name="ConnectStatus" >
             <property name="enabled" >
              <bool>true</bool>
             </property>
             <property name="palette" >
              <palette>
               <active>
                <colorrole role="WindowText" >
                 <brush brushstyle="SolidPattern" >
                  <color alpha="255" >
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Text" >
                 <brush brushstyle="SolidPattern" >
                  <color alpha="255" >
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Highlight" >
                 <brush brushstyle="SolidPattern" >
                  <color alpha="255" >
                   <red>181</red>
                   <green>213</green>
                   <blue>255</blue>
                  </color>
                 </brush>
                </colorrole>
               </active>
               <inactive>
                <colorrole role="WindowText" >
                 <brush brushstyle="SolidPattern" >
                  <color alpha="255" >
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Text" >
                 <brush brushstyle="SolidPattern" >
                  <color alpha="255" >
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Highlight" >
                 <brush brushstyle="SolidPattern" >
                  <color alpha="255" >
                   <red>62</red>
                   <green>146</green>
                   <blue>255</blue>
                  </color>
                 </brush>
                </colorrole>
               </inactive>
               <disabled>
                <colorrole role="WindowText" >
                 <brush brushstyle="SolidPattern" >
                  <color alpha="255" >
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Text" >
                 <brush brushstyle="SolidPattern" >
                  <color alpha="255" >
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Highlight" >
                 <brush brushstyle="SolidPattern" >
                  <color alpha="255" >
                   <red>62</red>
                   <green>146</green>
                   <blue>255</blue>
                  </color>
                 </brush>
                </colorrole>
               </disabled>
              </palette>
             </property>
             <property name="font" >
              <font>
               <family>Courier</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="whatsThis" >
              <string>This buffer displays configuration and connection information.</string>
             </property>
             <property name="verticalScrollBarPolicy" >
              <enum>Qt::ScrollBarAsNeeded</enum>
             </property>
             <property name="horizontalScrollBarPolicy" >
              <enum>Qt::ScrollBarAsNeeded</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="0" >
            <widget class="QLabel" name="label_4" >
             <property name="layoutDirection" >
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text" >
              <string>Logfile:</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="textInteractionFlags" >
              <enum>Qt::NoTextInteraction</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QLabel" name="outfileLabel" >
             <property name="toolTip" >
              <string>Logged output is written here.</string>
             </property>
             <property name="text" >
              <string>[none]</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2" >
       <attribute name="title" >
        <string>Sensors</string>
       </attribute>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="0" >
         <widget class="QTableWidget" name="sensorValues" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="palette" >
           <palette>
            <active>
             <colorrole role="Text" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="AlternateBase" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="Text" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="AlternateBase" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="Text" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="AlternateBase" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="autoFillBackground" >
           <bool>false</bool>
          </property>
          <property name="frameShape" >
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="lineWidth" >
           <number>1</number>
          </property>
          <property name="alternatingRowColors" >
           <bool>true</bool>
          </property>
          <property name="gridStyle" >
           <enum>Qt::DashLine</enum>
          </property>
          <property name="columnCount" >
           <number>2</number>
          </property>
          <column/>
          <column/>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="sensorProgressBar" >
      <property name="value" >
       <number>0</number>
      </property>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>499</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuControls" >
    <property name="title" >
     <string>Controls</string>
    </property>
    <addaction name="actionConnect" />
    <addaction name="actionDisconnect" />
   </widget>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionChoose_Log_Directory" />
    <addaction name="separator" />
    <addaction name="actionAbout" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuControls" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="toolButtonStyle" >
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="actionConnect" />
   <addaction name="actionDisconnect" />
   <addaction name="separator" />
   <addaction name="actionChoose_Log_Directory" />
  </widget>
  <action name="actionChoose_Log_Directory" >
   <property name="icon" >
    <iconset>:/icons/crystal_project/mydocuments.png</iconset>
   </property>
   <property name="text" >
    <string>Choose Log Directory</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="text" >
    <string>About</string>
   </property>
  </action>
  <action name="actionConnect" >
   <property name="icon" >
    <iconset>:/icons/crystal_project/usb.png</iconset>
   </property>
   <property name="text" >
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect" >
   <property name="icon" >
    <iconset>:/icons/nuoveXT-kde-1.6/error.png</iconset>
   </property>
   <property name="text" >
    <string>Disconnect</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tabControl</tabstop>
  <tabstop>PortComboBox</tabstop>
  <tabstop>BaudRateBox</tabstop>
  <tabstop>LogSpecComboBox</tabstop>
  <tabstop>specReloadButton</tabstop>
  <tabstop>ConnectStatus</tabstop>
  <tabstop>sensorValues</tabstop>
 </tabstops>
 <resources>
  <include location="MacLogger.qrc" />
 </resources>
 <connections/>
</ui>
